storagenode/piecestore/verification.go

```diff
@@ -42,7 +42,8 @@ func (endpoint *Endpoint) VerifyOrderLimit(ctx context.Context, limit *pb.OrderL
 		return ErrProtocol.New("piece expired: %v", limit.PieceExpiration)
 	case endpoint.IsExpired(limit.OrderExpiration):
 		return ErrProtocol.New("order expired: %v", limit.OrderExpiration)
-
+	case time.Now().UTC().Sub(limit.OrderCreation) > endpoint.config.OrderLimitGracePeriod:
```

Comment:
I don't understand the overuse of `UTC()` in our code base.

AFAIK, for every `t time.Time`, it is true that `t.Equal(t.UTC())`: https://play.golang.org/p/NNLpRtVd7Lm

So this line should work the same way without calling the `UTC()` method.

Reply:
lack of knowledge leads to overuse